lib/mock.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) IBM Corp. 2017.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from lib import config
+from lib import utils
+
+CONF = config.get_config().CONF
+
+class Mock(object):
+
+    def __init__(self, config_file, unique_extension):
+        """
+        Constructor
```

Comment:
Haha, gave it a meaningful description: `Initialize arguments common to all mock calls.`